app/models/metric/capture.rb

```diff
@@ -42,6 +42,11 @@ def self.perf_capture_timer(ems_id)
     _log.info("Queueing performance capture...")
 
     ems = ExtManagementSystem.find(ems_id)
+    if !ems.enabled?
```

Comment:
I'm fine with `if !` but I thought for sure the 🤖 🚓  would complain.  I think it reads better as a guard clause like this.

Reply:
Yeah I was pleasantly surprised I didn't get yelled at also 😁
I like unless at the end of a line but not as much in a standalone conditional like this